grass7/raster/r.centroids/r.centroids.html

```diff
@@ -0,0 +1,48 @@
+<p>
+<em>r.centroids</em> is a wrapper function for <a href="https://grass.osgeo.org/grass-stable/manuals/r.volume.html">r.volume</a>
+that computes the center point of raster clumps of data (neighboring, same-valued pixels). The centroids will always fall within
+the clump so they may not be the true, mathematical centroid. The centroids will always fall at the center of a pixel.
+<p>
+
+<h3>Options and flags:</h3>
+
+<em>r.centroids</em> requires an input raster containing clumps of data such as the 
+output of <a href="https://grass.osgeo.org/grass-stable/manuals/r.clump.html">r.clump</a>.
+
+<h2>EXAMPLES</h2>
+Find the centroids of the basins map (North Carolina sample dataset).
+
+First, set computational area.
+<div class="code"><pre>
+g.region raster=basin_50K
+</pre></div>
+
+Then, compute the centroids.
+<div class="code"><pre></pre>
+r.centroids input=basin_50K output=centroids50K
+</pre></div>
+
+<center>
+<a href="centroids_basins_50K.png">
+<img src="centroids_basins_50K.png" border="0" width="600" alt="r.centroids results">
```

Comment:
Please rename it to `r_centroids_basins_50K.png` (also the file) as per our file name convention.